requirements.txt

```diff
@@ -0,0 +1,42 @@
+ansible==2.5.2
```

Comment:
Why add the `requirements.txt`?

Reply:
I couldn't get this to run in pyvenv in an ide or locally with `python cli.py` until i had more stuff that wasn't in `setup.py`. I am not sure why this is but something is maybe weird with the setup. I can start fresh again but this was a force of habit on how I got my development env working.

Reply:
Hmm curious. I only worry about this `requirements.txt` getting out of date compared to the setup.py. 

I'll bet its a quirk of the env for your IDE. I don't mind leaving it if its necessary or helpful for development.

Reply:
I should try things again - is it possible to pip install the requirements but not the package with setup.py?

Reply:
ah - `pip install -e <path>`

Reply:
I got this sorted out I think - the pip install -e with symlinking seems to make things work as expected